Modules/Products/Feature/Modules/ProductsExportImportTest.php

```
@@ -0,0 +1,120 @@
+            $this->assertStringContainsString($product->name, $content);
+        }
+    }
+
+    #[Test]
+    #[Group('export')]
+    public function it_exports_products_downloads_excel_with_correct_data(): void
+    {
+        $this->markTestIncomplete();
+
+        /* Arrange */
+        $products = Product::factory()->for($this->company)->count(3)->create();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$products'. (undefined)

(UnusedLocalVariable)